entry/src/main/js/default/pages/index/index.css

```diff
@@ -0,0 +1,251 @@
+/* xxx.css */
+.body{
```

Comment:
Format all the code.
You can use keyboard shortcut Ctrl+Alt+L to do code formatting in Deveco IDE.